getpaid/utils.py

```diff
@@ -81,3 +94,17 @@ def qs_to_ordered_params(query_string):
             field = field.decode('utf8')
         params[field] = value
     return params
+
+
+def get_domain(request=None):
+    if (hasattr(settings, 'GETPAID_SITE_DOMAIN') and
+            settings.GETPAID_SITE_DOMAIN):
+        return settings.GETPAID_SITE_DOMAIN
+    if hasattr(settings, 'SITE_URL') and settings.SITE_URL:
```

Comment:
@kamilglod it will be confusing to use URL here because some of the backends build base url itself. How about we go with GETPAID_SITE_DOMAIN only instead?

@cypreess I think there were reasons for using solely Site, but it may be a good idea to include this? What do you think about it?